ci/features/step_definitions/profile_d_steps.rb

```diff
@@ -1,17 +1,10 @@
 When(/^the retrieve secrets \.profile\.d script is sourced$/) do
   @commands ||= []
   @commands << <<EOL
-. #{@BUILD_DIR}/.profile.d/0001_retrieve-secrets.sh #{@BUILD_DIR}
+HOME=#{@BUILD_DIR} DEPS_DIR=#{@DEPS_DIR} . #{@DEPS_DIR}/#{@INDEX_DIR}/.profile.d/0001_retrieve-secrets.sh
```

Comment:
Can we add a comment here explaining the reason for passing in HOME / DEPS_DIR and the syntax being used?

Reply:
Yep, added a comment.